keyboards/40percentclub/5x5/5x5.c

```diff
@@ -27,3 +27,14 @@ void led_set_kb(uint8_t usb_led) {
 
 	led_set_user(usb_led);
 }
+
+#ifdef SWAP_HANDS_ENABLE
+__attribute__ ((weak))
+const keypos_t hand_swap_config[MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
The 40percentclub/5x5 can be configured with 1 to 3 boards. If you were to use 2 and enable swap hands, it would not function as expected.

While not a common usage, would it be worth documenting the above behaviour somewhere?

Reply:
Well, leaving it weak should allow somebody to replace it, so it would be correct. 

However, I'm not sure that's the best way to handle this. Though, I'm not sure what way would be best.

Reply:
The main thing i took away from the change was that previously it would fail at compile time, where as now it it will behave oddly at run time. From an end user perspective, having a line output at compile time `Note: SWAP_HANDS_ENABLE its only configured for ortho_5x12` or the previously mentioned readme line could help.